src/Common/src/Interop/Windows/IpHlpApi/Interop.if_nametoindex.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class IpHlpApi
+    {
+        [DllImport(Interop.Libraries.IpHlpApi, SetLastError = true)]
+        internal extern static uint if_nametoindex(string name);
```

Comment:
It looks like this isn't available in UWP, which is why a few of the legs are failing with a BCL0015 error.  It'll need to be conditionally used when not building for that configuration.